tests/test_forecast/test_forecast.py

```diff
@@ -0,0 +1,209 @@
+from __future__ import unicode_literals
+
+import boto3
+import sure  # noqa
+
+from parameterized import parameterized
+from botocore.exceptions import ClientError
+from nose.tools import assert_raises
+
+from moto import mock_sts, mock_forecast
+from moto.core import ACCOUNT_ID
+from moto.forecast.models import DatasetGroup
+
+region = "us-east-1"
+account_id = None
+valid_domains = [
+    "RETAIL",
+    "CUSTOM",
+    "INVENTORY_PLANNING",
+    "EC2_CAPACITY",
+    "WORK_FORCE",
+    "WEB_TRAFFIC",
+    "METRICS",
+]
+
+
+@parameterized(valid_domains)
+@mock_forecast
+@mock_sts
```

Comment:
The `mock_sts` is not necessary for any of these tests.
(They were necessary for the StepFunctions example, which I guess you based this on, because SFN does some token validation that requires STS)

Reply:
Thanks - I wasn't sure if it was required or not; I've removed it and run tests again without issue.